src/stores/WalletStore.ts

```diff
@@ -38,24 +36,39 @@ class WalletStore extends PersistableStore {
     return web3Modal.cachedProvider
   }
 
+  private async getUserNetworkName() {
+    return (await provider.getNetwork()).name
+  }
+
+  private async isNetworkRight() {
```

Comment:
Let's place `private` methods near each other, we have a couple of them down below

Reply:
Will fix this, ty